workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityWorkerMenuGenerator.tt

```diff
@@ -0,0 +1,84 @@
+﻿<#@ template language="C#" #>
+<#@ output extension=".cs" #>
+<#
+    var generatedHeader = CommonGeneratorUtils.GetGeneratedHeader();
+    var workerTypeList = GetWorkerTypes();
+#>
+<#= generatedHeader #>
+
+#if UNITY_EDITOR
+using Improbable.Gdk.BuildSystem;
+using Improbable.Gdk.BuildSystem.Configuration;
+using Improbable.Gdk.Tools;
+using UnityEditor;
+using UnityEngine;
+
+namespace <#= qualifiedNamespace #>
+{ 
+    internal static class BuildWorkerMenu
```

Comment:
Why make this internal?

Reply:
because it's not needed anywhere else 🤷‍♀️